Gemfile

```diff
@@ -22,8 +22,10 @@ gem 'json'
 gem 'coffee-script'
 gem 'jquery-rails'
 gem 'prototype-rails' # FIXME: Will be needed with Rails3.1
-gem 'rugged'
+gem 'rugged', '~> 0.21.4'
```

Comment:
@david-yz-liu Sorry I should have elaborated... Having the rugged gem above version 0.21 was a requirement of the gitolite-rugged gem I added. https://github.com/oliverguenther/gitolite-rugged

That rugged gem was also over 2 years old, my spidey sense was tingling when I found out a gem the touched the file system was that outdated.